crystal_diffusion/data/diffusion/data_loader.py

```diff
@@ -0,0 +1,167 @@
+"""DataLoader from LAMMPS outputs for a diffusion model."""
+import logging
+import typing
+from dataclasses import dataclass
+from functools import partial
+from typing import Dict, Optional
+
+import datasets
+import pytorch_lightning as pl
+import torch
+import torch.nn.functional as F
+from torch.utils.data import DataLoader
+
+from crystal_diffusion.data.diffusion.data_preprocess import \
+    LammpsProcessorForDiffusion
+from crystal_diffusion.utils.hp_utils import check_and_log_hp
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass(kw_only=True)
+class LammpsLoaderParameters:
+    """Base Hyper-parameters for score networks."""
+
+    batch_size: int = 64
+    num_workers: int = 0
+    max_atom: int = 64
+    spatial_dimension: int = 3  # the dimension of Euclidean space where atoms live.
+
+
+class LammpsForDiffusionDataModule(pl.LightningDataModule):  # pragma: no cover
```

Comment:
Do we need the `#pragma: no cover`? Or is this an artifact from the cookie cutter?